ulog/sentry/sentry.go

```diff
@@ -70,22 +75,43 @@ func (c Configuration) Build() (zapcore.Core, error) {
 	if err != nil {
 		return zapcore.NewNopCore(), err
 	}
-	return newCore(client, zapcore.ErrorLevel), nil
+	return newCore(c, client, zapcore.ErrorLevel), nil
 }
 
 type core struct {
 	client
 	zapcore.LevelEnabler
 
-	fields map[string]interface{}
+	traceEnabled      bool
+	traceSkipFrames   int
+	traceContextLines int
```

Comment:
i am confused why these fields are needed. Don't use an unnamed struct in the config and use that here too?